.npmignore

```diff
@@ -0,0 +1,11 @@
+.babelrc.js
```

Comment:
ignoring some folders and files that dont need to published. by default npm also uses .gitignore file